routes/campaigns.js

```diff
@@ -328,14 +328,17 @@ router.get('/view/:id', passport.csrfProtection, (req, res) => {
 
             campaign.isScheduled = campaign.scheduled && campaign.scheduled > new Date();
 
+            // save total delivered count
+            campaing.orgdelivered = campaign.delivered
```

Comment:
@cold00water hey dude, think you missed the spelling mistake there :) `campaing` should be `campaign`